src/mpeg4/mpeg4Tag.ts

```diff
@@ -0,0 +1,30 @@
+import CombinedTag from "../combinedTag";
+import { Tag, TagTypes } from "../tag";
+
+export default class Mpeg4Tag extends CombinedTag {
+    /**
+     * Constructs and initializes a new instance with no contents.
+     */
+    public constructor() {
+        super(TagTypes.None, false);
+    }
+
+    public createTag(tagType: TagTypes, copy: boolean): Tag {
+        throw new Error("Method not implemented.");
+    }
+
+    /** @inheritDoc */
+    public addTag(tag: Tag): void {
```

Comment:
I'm kind of thinking we don't actually need these methods here. Though I guess it depends on whether the tags are built in or not. For example, ASF tag is part of the spec for ASF files so they can't be removed but can be empty. But if for MP4 they are optional, like RIFF files, it might make sense to add an implementation for the createTag method.

Reply:
They're not needed. In fact, the whole Mpeg4Box class should not exist (see comment above).

Reply:
Yup, see the comment above. Will close this one as probably irrelevant.